crate/src/page/partial/content_control_panel.rs

```diff
@@ -189,25 +189,25 @@ fn view_next_guide_link(next_guide: &Guide) -> impl View<Msg> {
                 ],
                 next_guide.menu_title,
             ],
-            view_next_icon().els(),
+            view_next_icon(),
         ]
     ]
 }
 
 // ------ view previous & next icon ------
 
-fn view_previous_icon() -> impl View<Msg> {
+fn view_previous_icon() -> Node<Msg> {
     div![
-        class![C.h_8, C.w_8,],
+        C![C.h_8, C.w_8,],
         style! {
             St::Transform => "rotate(180deg)",
         },
-        image::next_icon_svg().els()
+        image::next_icon_svg().into_nodes()
     ]
 }
 
-fn view_next_icon() -> impl View<Msg> {
-    div![class![C.h_8, C.w_8], image::next_icon_svg().els()]
+fn view_next_icon() -> Node<Msg> {
+    div![C![C.h_8, C.w_8], image::next_icon_svg().into_nodes()]
```

Comment:
Please change all `*_svg` functions to return `Vec<Msg>` so we can delete all `.into_nodes()` calls.